paasta_tools/cli/cmds/spark_run.py

```diff
@@ -1150,6 +1155,40 @@ def paasta_spark_run(args):
         needs_docker_cfg=needs_docker_cfg,
         auto_set_temporary_credentials_provider=auto_set_temporary_credentials_provider,
     )
+
+    # Mass enable DRA config
```

Comment:
* Add detailed doc: Why this is being added? Design, purpose, rollout plan.
* Ticket to rollback/remove once the migration is done

Reply:
Added description